collectors/ioping.plugin/metadata.yaml

```diff
@@ -1,54 +1,81 @@
+# yamllint disable rule:line-length
+---
 meta:
   plugin_name: ioping.plugin
   module_name: ioping.plugin
   monitored_instance:
     name: IOPing
-    link: ''
+    link: "https://github.com/koct9i/ioping"
     categories:
     - data-collection.synthetic-checks
-    icon_filename: 'syslog.png'
+    icon_filename: "syslog.png"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
+    description: ""
   keywords: []
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Monitor IOPing metrics for efficient disk I/O latency tracking. Keep track of read/write speeds, latency, and error rates for optimized disk operations.'
-    method_description: ''
+    metrics_description: "Monitor IOPing metrics for efficient disk I/O latency tracking. Keep track of read/write speeds, latency, and error rates for optimized disk operations."
+    method_description: "Plugin uses `ioping` command."
   supported_platforms:
     include: []
     exclude: []
   multi_instance: true
   additional_permissions:
-    description: ''
+    description: ""
   default_behavior:
     auto_detection:
-      description: ''
+      description: ""
     limits:
-      description: ''
+      description: ""
     performance_impact:
-      description: ''
+      description: ""
 setup:
   prerequisites:
-    list: []
+    list:
+      It is necessary to have the command `ioping` installed on your host. You can install the command by passing the argument `install` to the plugin:
```

Comment:
@thiagoftsm you need configuring schema validation in your IDE. It is in "Languages & Frameworks"->"Schemas and DTDs"->"JSON Schema Mappings"

Reply:
Vscode did not report issues at all.